DSCResources/MSFT_xADDomain/MSFT_xADDomain.schema.mof

```diff
@@ -1,13 +1,14 @@
 [ClassVersion("1.0.1.0"), FriendlyName("xADDomain")]
 class MSFT_xADDomain : OMI_BaseResource
 {
-    [Key] String DomainName;
-    [write] String ParentDomainName;
-    [write] String DomainNetbiosName;
-    [Required, EmbeddedInstance("MSFT_Credential")] String DomainAdministratorCredential;
-    [Required, EmbeddedInstance("MSFT_Credential")] String SafemodeAdministratorPassword;
-    [write,EmbeddedInstance("MSFT_Credential")] String DnsDelegationCredential;
-    [write] String DatabasePath;
-    [write] String LogPath;
-    [write] String SysvolPath;
+    [Key, Description("Name of the domain to which the user will be added")] String DomainName;
+    [Required, Description("Credentials used to query for domain existence"), EmbeddedInstance("MSFT_Credential")] String DomainAdministratorCredential;
+    [Required, Description("Password for the administrator account when the computer is started in Safe Mode"), EmbeddedInstance("MSFT_Credential")] String SafemodeAdministratorPassword;
+    [Write, Description("Fully qualified name of the parent domain")] String ParentDomainName;
+    [Write, Description("NetBIOS name for the new domain")] String DomainNetbiosName;
+    [Write, Description("Credential used for creating DNS delegation"), EmbeddedInstance("MSFT_Credential")] String DnsDelegationCredential;
+    [Write, Description("Path to a directory that contains the domain database")] String DatabasePath;
+    [Write, Description("Path to a directory for the log file that will be written")] String LogPath;
+    [Write, Description("Path to a directory where the Sysvol file will be written")] String SysvolPath;
+    [Write, Description("Specifies the DNS Server service should be not installed")] Boolean ExcludeDns;
```

Comment:
The property should describe state not just an installation property.  
Can we describe the state the installation property results in, instead of just copying the property?